docs/blog/newsletter-june-2023.rst

```diff
@@ -0,0 +1,128 @@
+.. post:: June 06, 2023
+   :tags: newsletter
+
+#####################################
+Write the Docs Newsletter – June 2023
+#####################################
+
+Shalom, documentarians! This month marks a year since I first started putting together this newsletter and it's been a great chance for me to learn morea bout all kinds of topics. If you've enjoyed the newsletter, help me learn even more by reacting the most insightful posts in Slack with the ``:suggest-for-newsletter:`` emoji. Although space is limited each month, I always enjoy seeing what everyone is learning from.
```

Comment:
```suggestion
Shalom, documentarians! This month marks a year since I first started putting together this newsletter and it's been a great chance for me to learn more about all kinds of topics. If you've enjoyed the newsletter, help me learn even more by reacting to the most insightful posts in Slack with the ``:suggest-for-newsletter:`` emoji. Although space is limited each month, I always enjoy seeing what everyone is learning from.
```

Reply:
```suggestion
Shalom, documentarians! This month marks a year since I first started putting together this newsletter and it's been a great chance for me to learn more about all kinds of topics. If you've enjoyed the newsletter, help me learn even more by reacting to the most insightful posts in Slack with the ``:suggest-for-newsletter:`` emoji. Although space is limited each month, I always enjoy seeing what everyone is learning.
```